curvefs/src/client/inode_wrapper.h

```diff
@@ -413,6 +419,9 @@ class InodeWrapper : public std::enable_shared_from_this<InodeWrapper> {
 
     mutable ::curve::common::Mutex syncingVolumeExtentsMtx_;
     ExtentCache extentCache_;
+
+    // timestamp when put in cache
+    uint64_t time_;
```

Comment:
I think `time_` should not be a property of `InodeWrapper`, but I didn't found another easier way.

Reply:
okay, let it stay here for now